### PR TITLE
docs: Improved styling around italic and corrected grammar.

### PR DESCRIPTION
**Pull request by MichaelJolley**
Nothing major here.  Loving Gridsome so hope to provide better PR's to documentation as I can.

### CODE REVIEW

File: docs/layouts.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Layout components are used to wrap pages. Layouts should contain components like headers, footers or sidebars that will be used across the site.
 
-Layouts are _just_  **.vue components** located in `src/layouts` and need to be [declared as a global](#make-a-layout-global) component or imported per page to be used.
+Layouts are _just_ &nbsp;**.vue components** located in `src/layouts` and need to be [declared as a global](#make-a-layout-global) component or imported per page to be used.
```
Comment (12vanblart) [inline docs/layouts.md]: It looks like this is adding a double space between 'just' and '.vue'?

Comment (12vanblart) [reply docs/layouts.md]: I take that back - I see what this is addressing now. It looks like GitHub is processing the markdown slightly differently than Gridsome is. 

Good catch!

Comment (MichaelJolley) [reply docs/layouts.md]: It is 100% for readability.  Without the `&nbsp;` the docs render as:

image

With the added `&nbsp;` they render as:

image

**Comment by mtlynch** [comment]
Thanks for these fixes @MichaelJolley and the review @12vanblart!

**Pull request merged**